packages/framer-scroller/hooks/useScrollTo.ts

```diff
@@ -34,7 +34,9 @@ export function useScrollTo() {
               },
               onComplete,
               onStop: onComplete,
-              duration: duration * 1000 || distanceAnimationDuration(ref.scrollLeft, to.x),
+              type: 'spring',
+              bounce: 0,
+              duration: duration * 2000 || distanceAnimationDuration(ref.scrollLeft, to.x),
```

Comment:
```suggestion
              duration: duration * 1000 || distanceAnimationDuration(ref.scrollLeft, to.x),
```

Reply:
All durations have to be adjusted if we go this route. Set it to 2000 for now to test.

Reply:
I dont think so, the duration is almost never provided and will almost always use the distanceAnimationDuration